meta/applicator.json

```diff
@@ -12,7 +12,10 @@
         "prefixItems": { "$ref": "#/$defs/schemaArray" },
         "items": { "$dynamicRef": "#meta" },
         "contains": { "$dynamicRef": "#meta" },
-        "additionalProperties": { "$dynamicRef": "#meta" },
+        "additionalProperties": {
+            "$dynamicRef": "#meta",
+            "default": {}
```

Comment:
Instead of adding a default in every keyword that uses a subschema, I would just add it to the top level schema definition right at `schema.json#meta`. (And `true` might be a more intuitive default than `{}`, at any rate.)

Reply:
Right, but the new spec (and probably the old because this likely didn't change) says:

> Omitting this keyword has the same assertion behavior as an empty schema.

Thus the default needs to be "an empty schema" which is _technically_ different from (although equivalent to) `true`.